data/transactions/logic/eval.go

```diff
@@ -49,36 +49,47 @@ const EvalMaxScratchSize = 255
 // MaxStringSize is the limit of byte strings created by `concat`
 const MaxStringSize = 4096
 
-// stackValue is the type for the operand stack.
-// Each stackValue is either a valid []byte value or a uint64 value.
-// If (.Bytes != nil) the stackValue is a []byte value, otherwise uint64 value.
-type stackValue struct {
+// StackValue is the type for the operand stack.
+// Each StackValue is either a valid []byte value or a uint64 value.
+// If (.Bytes != nil) the StackValue is a []byte value, otherwise uint64 value.
+type StackValue struct {
```

Comment:
it looks like no need to name `StackValue` exported

Reply:
Strange, I could've sworn I reverted that rename.

Reply:
Any tools like `go vet` that could help us find things like this?  Names we export for no reason?